.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      prerelease:
+        description: "Prerelease"
+        required: true
+        default: false
+        type: boolean
+      draft:
+        description: "Draft"
+        required: true
+        default: false
+        type: boolean
+      version-increment-type:
+          description: 'Which part of the version to increment:'
+          required: true
+          type: choice
+          options:
+            - major
+            - minor
+            - patch
+          default: 'patch'
+
+permissions:
+  contents: write
+  # packages: write
```

Comment:
if these are commented out could we just remove them?

Reply:
I was about to remove them, but then I thought they might help somebody down the road to remember that they might need to enable them.
e.g. if we decide to build and upload a docker container to GitHub packages.